q2_sample_classifier/plugin_setup.py

```diff
@@ -639,6 +640,22 @@
                 'displays (normalized) feature abundances per sample.'
 )
 
+plugin.methods.register_function(
+    function=shapely_values,
+    inputs={**inputs, 'sample_estimator': SampleEstimator[Classifier]},
+    parameters={},
+    outputs=[('shap', SampleData[Probabilities])],
+    input_descriptions={
+        'table': input_descriptions['table'],
+        'sample_estimator': 'Sample classifier trained with fit_classifier.'},
+    output_descriptions={
+        'shap': 'Contributions of each feature towards the prediction.'},
+    name='Use trained classifier to compute Shapely values new samples.',
```

Comment:
This name is a bit confusing to me. What do you mean with "new samples" at the end?

Reply:
I just changed that to all samples.